### PR TITLE
Fix Jenkins builds

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

**Comment by ace-n** [comment]
N.B: Jenkins is (currently) configured to pull from `master`, so its builds will be broken until this is merged.

File: package.json [hunk 1]
```diff
@@ -39,10 +39,17 @@
     ]
   },
   "dependencies": {
+    "@google/cloud-debug": "^0.8.3",
+    "@google/cloud-trace": "^0.5.5",
+    "@google-cloud/datastore": "^0.1.1",
+    "@google-cloud/storage": "^0.1.1",
+    "@google-cloud/pubsub": "^0.2.0",
```
Comment (jmdobry) [inline package.json]: Why are all of these dependencies needed at the root of the repo?

File: 1-hello-world/app.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ app.get('/', (req, res) => {
 if (module === require.main) {
   // [START server]
   // Start the server
-  const server = app.listen(process.env.PORT || 8080, () => {
+  const server = app.listen(process.env.PORT || 8081, () => {
```
Comment (jmdobry) [inline 1-hello-world/app.js]: I don't think the right solution is to manually hard code each step of the tutorial to a different port for the sake of the tests. Regardless of which step a user is working on, the app should listen on 8080 without any intervention from the user.

File: optional-container-engine/test/config.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ module.exports = {
   cwd: path.resolve(path.join(__dirname, `../`)),
   cmd: `node`,
   args: [`app.js`],
+  url: `http://localhost:8088`,
```
Comment (jmdobry) [inline optional-container-engine/test/config.js]: I think the correct solution is to change this line to `port: 8088` and modify the repo tools to dynamically set the `PORT` environment variable for the each subprocess that gets spawned for each deployment. This way each app can default to `8080`, but during the tests each app will use a different port based on the `PORT` env var in its subprocess.

File: package.json (hunk 1 above)
Comment (ace-n) [inline package.json]: @jmdobry Is this still an issue after our discussion about unifying dependencies yesterday?

**Pull request merged**